src/extensions/extension-loader.ts

```diff
@@ -81,28 +101,8 @@ export class ExtensionLoader extends Singleton {
     return extensions;
   }
 
-  @computed get userExtensionsByName(): Map<string, LensExtension> {
-    const extensions = new Map();
-
-    for (const [, val] of this.instances.toJSON()) {
-      if (val.isBundled) {
-        continue;
-      }
-
-      extensions.set(val.manifest.name, val);
-    }
-
-    return extensions;
-  }
-
-  getExtensionByName(name: string): LensExtension | null {
```

Comment:
This function as is doesn't do what you need? Did you have to add the `instancesByName` observable.map?

Reply:
No, it was an optimization. Similar to those done in `CatalogEntityRegistry` in `renderer`. To avoid rebuilding the whole map everytime a dependent is changed.